apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-inserter-inline-search-term.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+/**
+ * External dependencies
+ */
+import { debounce, get } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { select } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import tracksRecordEvent from './track-record-event';
+
+/**
+ * Handler function which takes over tracking blocks-searching events
+ * from the "Autocomplete / Block" component.
+ * It worths mentioning that the running of it
+ * will be conditioned by a valid target,
+ * which is previously handled by the `selectorHandler()` function.
+ * The most important checks are performed there.
+ */
+const trackInserterInlineSearchTerm = () => {
+	// Pick up the search term from the `content` block attributes.
+	const search_term = get(
+		select( 'core/block-editor' ).getSelectedBlock(),
+		[ 'attributes', 'content' ],
+		''
+	).substr( 1 );
+
+	if ( search_term.length < 3 ) {
+		return;
+	}
+
+	const context = 'inserter_inline';
+
+	tracksRecordEvent( 'wpcom_block_picker_search_term', {
+		search_term,
+		context,
+	} );
```

Comment:
@martinremy Should we introduce a new event that encompasses search events in both the block picker and the inline `/` command? Or continue with `'wpcom_block_picker_search_term'` and add context to it to denote the difference?

Reply:
we will be able to segment the result by context, in a similar way that we do with user_locale and/or selected_block.

![image](https://user-images.githubusercontent.com/77539/79274610-3300f780-7e7b-11ea-8335-f152e904b536.png)

_Keep in mind there are some dirty data produced by the development process. The only one context values will be `inserter_menu` and `inserter_inline` so far._

Reply:
I like using the same event and distinguishing with the context property.  As an aside, I don't why the `_term` is there at the end of `wpcom_block_picker_search_term` eventname, since it's basically fired for a search, right?  But that's not important, just an OCD thing.

Reply:
Thank you!